CONTRIBUTING.md

```diff
@@ -43,7 +43,7 @@ development of the library, from your local clone of the fork, run
 
 ```bash
 (myenv) pip install -e .
-(myenv) pip install -r requirements.txt
+(myenv) pip install -r dev_requirements.txt
```

Comment:
Is it somewhere clear that you also need to `pip install -r requirements.txt`?

Reply:
Also isn't `make install` a thing? Do we want to say that here? @nathanshammah

Reply:
> Is it somewhere clear that you also need to `pip install -r requirements.txt`?

Not really, except for here which was replaced to be `dev_requirements`. I don't think there's really a need to specify `requirements` because whenever I have tested the minimal qiskit install, all of the packages from `requirements` were already installed after `mitiq` was installed using `pip install -e .`